packages/desktop-client/src/components/accounts/MobileAccountDetails.js

```diff
@@ -0,0 +1,180 @@
+import React, { useMemo } from 'react';
+import { Link } from 'react-router-dom';
+
+import { Label, View, Input, Button } from 'loot-design/src/components/common';
+import CellValue from 'loot-design/src/components/spreadsheet/CellValue';
+import Text from 'loot-design/src/components/Text';
+import { colors, mobileStyles as styles } from 'loot-design/src/style';
+import Add from 'loot-design/src/svg/v1/Add';
+import CheveronLeft from 'loot-design/src/svg/v1/CheveronLeft';
+import Search from 'loot-design/src/svg/v1/Search';
+
+import { TransactionList } from './MobileTransaction';
+
+class TransactionSearchInput extends React.Component {
```

Comment:
These are all copy/pasted from the React Native package, then tweaked as little as possible to get them working. 
In case you missed it, the PR description goes into a lot of detail about how I approached these components. For instance, leaving lines commented rather than removed so folks can see how currently disabled behaviors used to be implemented in RN without a lot of digging in packages/mobile.

Reply:
Ouch, that's pretty nasty.. these two duplicates will just diverge more and more as people work here and add features/fix issues. How can we improve the situation?

IMO the long-term plan is to [kill mobile apps](https://actualbudget.github.io/docs/FAQ/) (and thus also the code). So we could treat this new code as the source-of-truth. Which then also means we can remove the hacks, comments etc. from this new version.

Reply:
Oh yeah, I hear ya. Too much to try to dive into thoroughly during a work break today, but I think it gets into our long-term strategy for how to do responsiveness. 

At the time I whipped this out back in July, I assumed we wanted to essentially port the whole native app.  That's not the direction @jlongster had in mind, last we chatted. We said we'd like to find a way to unify the components that I've split out here, in which case none of the old patterns from the React Native apps will apply anymore and the unused bits here will be 100% junk. It's just been difficult (for me, anyway) to imagine some of the keyboard-heavy components with desktop-specific UX patterns morphing seamlessly to mobile in what I think of as traditional responsiveness.

But...if we decide to take the RN port any further (say, to have MVP mobile transaction entry in the near term), it seems easier for contributors to get up to speed if they can see how things were done in the native app that long-time users are familiar with without an archaeological expedition to `packages/mobile`. 

I dunno, it all feels different to me when you have a team of dedicated devs who know the codebase inside-and-out vs volunteers trying to make progress in limited free time. 

If everybody prefers tidy, though, I can definitely strip out the unused stuff like I would in my day job. Speaking of...gotta run... :D